stream_consumer.go

```diff
@@ -20,6 +20,34 @@ import (
 	"time"
 )
 
+const (
+	StreamConsumerReceivedEvents         = "stream_consumer_received_events"
+	StreamConsumerConnectionAttempts     = "stream_consumer_connection_attempts"
+	StreamConsumerConnectionStatusChecks = "stream_consumer_connection_status_checks"
+	StreamConsumerConnectionStatus       = "stream_consumer_connection_status"
+	StreamConsumerConnectionSuccess      = "stream_consumer_connection_success"
+	StreamConsumerConnectionFailure      = "stream_consumer_connection_failure"
+	StreamConsumerDisconnections         = "stream_consumer_disconnections"
+	StreamConsumerConnected              = "stream_consumer_connected"
+	StreamConsumerDelayMs                = "stream_consumer_delay_ms"
+	StreamConsumerOriginDelayMs          = "stream_consumer_origin_delay_ms"
+	StreamConsumerEventDelayMs           = "stream_consumer_event_delay_ms"
+)
+
+const StreamEndpointsLabel = "endpoints"
+
+func IsStreamConsumerMetric(m string) bool {
```

Comment:
Where is this used?

Reply:
Or, asked differently: why do we manage differently stream metrics?

Reply:
This is used in the monitoring to filter and display only stream metrics.
I put it there to keep it close to the constants definition, but this could also be moved in the monitoring component since it will probably not be used by any other component.